fix(menu-toggle): fix split button inline padding

mcoker opened the pull request:
fixes https://github.com/patternfly/patternfly/issues/7197

This was originally in https://github.com/patternfly/patternfly/pull/7138, but I closed that PR since it needed more work to fix a separate issue and @sg00dwin is taking the lead on that.

From https://github.com/patternfly/patternfly/pull/7138#pullrequestreview-2341771216, this is the idea behind this fix

> This was just targeting the wrong element. Needs to look for `__controls` as an only child (meaning no `__text` sibling). It's effectively the same style as this one, but in `:not()` to target the inverse https://github.com/patternfly/patternfly/blob/689a71891bac6823c3dc2bd1406ea713f7037e3d/src/patternfly/components/MenuToggle/menu-toggle.scss#L486

patternfly-build commented:
Preview: https://patternfly-pr-7198.surge.sh

sg00dwin commented:
/lgtm

sg00dwin reviewed (approved):
/lgtm

sg00dwin commented:
@mcoker  the `menu-toggle` variants at small sizes, the split toggle examples have inconsistencies with the small left and right padding, on both the `pf-v6-c-menu-toggle__button` and `label`, when comparing to the regular small buttons.

![Screenshot 2024-11-18 at 9 34 06 AM](https://github.com/user-attachments/assets/1891728a-fd02-4a1e-a7dd-c857712c2695)
---

![Screenshot 2024-11-18 at 9 43 46 AM](https://github.com/user-attachments/assets/91654541-c839-4779-a0f5-940f115efa1f)
![Screenshot 2024-11-18 at 9 32 41 AM](https://github.com/user-attachments/assets/1c8904b7-6b79-486a-b58c-2f4c1c84df05)
![Screenshot 2024-11-18 at 9 31 38 AM](https://github.com/user-attachments/assets/dd37fd6f-7403-4a48-8a7a-6429d0262912)
![Screenshot 2024-11-18 at 9 30 35 AM](https://github.com/user-attachments/assets/11485320-ad5b-469f-807c-ba44485fa629)
![Screenshot 2024-11-18 at 9 37 17 AM (2)](https://github.com/user-attachments/assets/19f118a7-e820-426a-96fb-206b5b74878d)

mcoker commented:
@sg00dwin nice catch! Like we discussed offline, I updated the small button variations to use the correct semantic spacers and the menu-toggle. Also added a few more toggles to the "small" menu toggle examples. Here's a screenshot of what all of the split button toggles look like when set to small.

<img width="1624" alt="Screenshot 2024-11-19 at 9 30 02 PM" src="https://github.com/user-attachments/assets/c9d738dc-993c-463a-a3bc-42dba7494bbc">
<img width="1624" alt="Screenshot 2024-11-19 at 9 29 55 PM" src="https://github.com/user-attachments/assets/4c01757e-9d32-486f-8f50-7dbc8ff0bfaf">

@lboehling wdyt? The small buttons with only a checkbox look a little cramped, but it's the same spacing scheme as non-small - they use control spacers all around.

sg00dwin reviewed (approved):
/lgtm

srambach reviewed (approved):
Code LGTM 🦃

lboehling reviewed (approved):
lgtm!

patternfly-build commented:
:tada: This PR is included in version 6.1.0-prerelease.9 :tada:

The release is available on:
- [GitHub release](https://github.com/patternfly/patternfly/releases/tag/prerelease-v6.1.0-prerelease.9)
- [npm package (@prerelease dist-tag)](https://www.npmjs.com/package/@patternfly/patternfly/v/6.1.0-prerelease.9)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
